docs/content/Coding Milestones/PoC2023q1/commands.md

```diff
@@ -761,37 +767,28 @@ Creating or updating cluster role binding "kubestellar-syncer-demo2-15nq4e94" to
 
 Wrote WEC manifest to demo2-syncer.yaml for namespace "kubestellar-syncer-demo2-15nq4e94". Use
 
-  KUBECONFIG=<workload-execution-cluster-config> kubectl apply -f "demo2-syncer.yaml"
+  KUBECONFIG=<space-execution-cluster-config> kubectl apply -f "demo2-syncer.yaml"
 
 to apply it. Use
 
-  KUBECONFIG=<workload-execution-cluster-config> kubectl get deployment -n "kubestellar-syncer-demo2-15nq4e94" kubestellar-syncer-demo2-15nq4e94
+  KUBECONFIG=<space-execution-cluster-config> kubectl get deployment -n "kubestellar-syncer-demo2-15nq4e94" kubestellar-syncer-demo2-15nq4e94
 
 to verify the syncer pod is running.
 ```
 
-## Creating a Workload Management Workspace
+## Creating a Workload Management Space
```

Comment:
Do not go half-way. Either keep it "Workload Management Workspace" or change it to "Workload Description Space".